Makefile

```
@@ -72,15 +74,17 @@ format-python: | $(VENV)
 	; done
 
 generate-api-docs:
-	npx openapi-generator generate -i ./templates/api.definition.yml -g markdown -t ./docs/templates/ -o docs/api
+	$(eval TEMP_FILE := $(shell mktemp))
+	cat ./templates/api.yaml | $(VENV)/bin/yq -y .Resources.Api.Properties.DefinitionBody > $(TEMP_FILE)
```

```suggestion
	$(VENV)/bin/yq -y .Resources.Api.Properties.DefinitionBody ./templates/api.yaml > $(TEMP_FILE)
```

Remove unnecessary use of cat :)

---

Thank you. I also made one more change here where is did the same thing with TEMP_FILE variable as you suggested for the sed command